mail_template_default/__manifest__.py

```diff
@@ -0,0 +1,16 @@
+# Copyright 2023 Solvti sp. z o.o. (https://solvti.pl)
+
+{
+    "name": "Mail Template Default",
```

Comment:
Probably, the name should be changed, to avoid misunderstanding.
This addon adds the rules, not default template only for a wizard. You can simply add Rules at the end?